src/pages/profile/index.js

```diff
@@ -1,24 +1,28 @@
 import React from 'react'
 import { Redirect, useHistory } from 'react-router-dom'
-import { useForm } from 'react-hook-form'
 import { useDispatch, useSelector } from 'react-redux'
+import { useForm } from 'react-hook-form'
 import { isEmpty } from 'lodash'
 
+import routes from '../../routes'
 import { setFilms } from '../../api/films'
-import Layout from '../../components/Layout'
 import { deleteUser } from '../../api/user'
-import routes from '../../routes'
+import { img, name, text } from '../../constans'
```

Comment:
there is naming convention for naming constants
it goes like `IMG`, `NAME`, `TEXT`

but this is still not acceptable names for constants and variables
you should pick more distinct names for this constants
more like `PROFILE_IMAGE_FIELD`, `PROFILE_NAME_FIELD`, etc